### DISCUSSION TITLE
Feature request: Add Prev/Next buttons to the bottom of a commit screen

### DISCUSSION THREAD

**Discussion by rodrigoescobar** [community/community#69849] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

Hello PR team,

In my team, we spend a decent amount of time structuring our PRs to make it easier for the reviewer to follow the changes and therefore we structure it so that each commit tells a part of the story.

As we review our PRs commit-by-commit, I find myself in the annoying situation of having to go all the way back to the top of the screen to click on the `Next` button to go to the next commit. This is specially painful when a commit is simple but touches many files and therefore is quite long.

 I would appreciate if we could add these buttons to the bottom of the commit itself, please.

**Comment by yguedidi** [comment]
An alternative solution could be to have a simplified version of the commit info block that contains prev/next action be sticky while scrolling.
I would suggest at least have the commit summary so that it preserve context and the prev/next actions for easy access
